src/generator/navigation.clj

```diff
@@ -35,7 +35,9 @@
     [:nav {:class "navigation"} 
      [:div {:class "nav-wrapper"}
       [:a {:href (create-url "/") :class "logo"}
-       [:img {:src (static/get-asset-url "images/cartman_digital_logo.svg") :title "Cartman Digital"}]
+       [:picture
+        [:source {:srcset "/assets/images/cartman_logo_purple_white.png" :media "(prefers-color-scheme: dark)"}]
+        [:img {:src "/assets/images/cartman_digital_logo_dark.png"}]]
```

Comment:
Please use static/get-asset-url to get the urls here. otherwise you do not get the asset version number into the url.